httpapi/v2/documents_api.go

```diff
@@ -82,6 +89,59 @@ func (h handler) CreateDocument(w http.ResponseWriter, r *http.Request) {
 	render.JSON(w, r, resp)
 }
 
+// CloneDocument creates a new cloned document from an existing Template document.
+// @summary Creates a new cloned document from an existing Template document.
+// @description Creates a new cloned document from an existing Template document.
+// @id clone_document_v2
+// @tags Documents
+// @accept json
+// @param authorization header string true "Hex encoded centrifuge ID of the account for the intended API action"
+// @param body body v2.CloneDocumentRequest true "Document Clone request"
+// @produce json
+// @Failure 400 {object} httputils.HTTPError
+// @Failure 500 {object} httputils.HTTPError
+// @Failure 403 {object} httputils.HTTPError
+// @success 201 {object} coreapi.DocumentResponse
+// @router /v2/documents/clone [post]
```

Comment:
IMO the url should be of the following form
`v2/documents/{doc_id}/clone`
Indicates we are cloning `docID`
WDYT ?

Reply:
yes, I think it makes sense